src/components/ForgotPassword/ForgotPassword.js

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { injectIntl } from 'react-intl';
+import { timeout } from '../../utils';
+import { Formik, Form } from 'formik';
+import { FieldGroup, InputFieldItem } from '../form-helpers/form-helpers';
+
+const fieldNames = {
+  email: 'email'
+};
+
+/** Prepare empty values for all fields
+ * It is required because in another way, the fields are not marked as touched.
+ */
+const getFormInitialValues = () =>
+  Object.keys(fieldNames).reduce(
+    (accumulator, currentValue) => ({ ...accumulator, [currentValue]: '' }),
+    {},
+  );
+
+const ForgotPassword = ({ intl }) => {
+
+  const _ = (id, values) => intl.formatMessage({ id: id }, values);
+
+  const validate = (values) => {
+    const errors = {};
+
+    if (!values[fieldNames.email]) {
+      errors[fieldNames.email] = _('validation_messages.error_required_field');
+    } else if (!/^[A-Z0-9._%+-]+@[A-Z0-9.-]+\.[A-Z]{2,4}$/i.test(values[fieldNames.email])) {
```

Comment:
Maybe add this regex in a more reusable place? you have password validation in signup and here.

Reply:
You are right, although this validation is still pending, there is a task to deal with these things: [DBR-135](http://jira.makingsense.com/browse/DBR-135)

Reply:
Yes, but there is another task for validations